PENDING_RELEASE_NOTES.md

```diff
@@ -5,6 +5,7 @@ _Date TBD_
 ### New Features
 * Add FF4j framework (feature flags)
 * Add endpoints to provide ability for users to save filters for admin reports
+* Add endpoints to provide activity metadata for users, announcements, pending listings, corrective action plans, and pending surveillances
```

Comment:
Was there supposed to be API key activity here too? Should we have a metadata endpoint for corrective action plans since they are no longer part of the system? There are some old ones so I don't really know the answer to this, just asking the question.